mods/rv/rules/ai.yaml

```diff
@@ -758,6 +780,10 @@ Player:
 			commanders_power.cruiser_strike: 1
 			commanders_power.v3_strorm: 1
 			commanders_power.orbital_drop: 1
+		UnitLimits:
+			cmin: 6
+			harv: 6
+			gmin: 6
```

Comment:
It is a bug that this was lacking the unit limits, can you add the ones from Megawealth version too.

Reply:
it is a little bit senseless for 8 harvesters because mines in RV running out quickly, and AI won't find new mine ( use harvester bot module can let them go to new mine but it is a ridiculous performance eater when it looking for mine, meanwhile it is helpless to AI economy in remote mining). 

I suggest only build 6 harvester and give AI cash cheat for every economic building built. This can make AI have a relatively reasonable economy while avoid tangling with the unfinished subbase & harvesters economy for AI.

Reply:
Sounds good to me, 6 it is then.